packages/mentions-wrapper/src/UserList.js

```diff
@@ -47,6 +48,8 @@ export const UserList = ({
 
         onSelect(user);
     };
+    
+    const sortedUsers = sortBy(users, [userName => userName.displayName]);
```

Comment:
Consider moving this to line 75 rather than defining a var sortedUsers. There is no point calling sortBy if users is an empty array.